spec/cc/analyzer/path_entries_spec.rb

```diff
@@ -0,0 +1,21 @@
+require "spec_helper"
+
+module CC::Analyzer
+  describe PathPatterns do
+    include FileSystemHelpers
+
+    around do |test|
+      within_temp_dir { test.call }
+    end
+
+    describe "#entries" do
+      it "filters out trailing slashes" do
+        make_file("lib/foo.rb")
+
+        paths = PathEntries.new("lib/").entries
+
+        paths.sort.must_equal(["lib", "lib/foo.rb"])
+      end
```

Comment:
Nice! I like this test. Can we also have a test that checks what happens when passed dir with no slash (e.g. `lib`), and multiple paths `lib test.rb app/models`?